### PR TITLE
Black border around notification indicator fixed

### LINKED ISSUE #4: Black Border on notification indicator
The border on blue notification indicator circle icon placed over the notification icon is still black
screenshot from 2017-02-20 12-15-54

### PR DESCRIPTION
**Pull request by anshulsahni**
#4 fixed

### CODE REVIEW

**Comment by victor-li** [comment]
Hi @anshulsahni,

Thank you for your contribution!
In the previous version of the GitHub CSS, the following code was used:
```css
.notification-indicator .mail-status {
    ...
    border: 2px solid #f3f3f3;
    ...
}
```

Could you change it into that? Afterwards, I will merge the change.
Thanks! 😄

File: GitHubWhiteNavBar.user.js [hunk 1]
```diff
@@ -72,4 +72,7 @@ GM_addStyle('\
     color: #4078c0;\
     border-right-color: #c6d7ec;\
 }\
+.notification-indicator .mail-status {\
+    border: none;\
```
Comment (victor-li) [inline GitHubWhiteNavBar.user.js]: Please change this into `border: 2px solid #f3f3f3;`

**Review by victor-li — CHANGES_REQUESTED** [review]
Hi @anshulsahni,

Thank you for your contribution!
In the previous version of the GitHub CSS, the following code was used:
```css
.notification-indicator .mail-status {
    ...
    border: 2px solid #f3f3f3;
    ...
}
```

Could you change it into that? Afterwards, I will merge the change.
Thanks! 😄

**Comment by anshulsahni** [comment]
@victor-li  fixed

screenshot from 2017-02-21 13-15-34

**Comment by victor-li** [comment]
Thanks @anshulsahni! 😄

**Pull request merged**